Source/RuntimeFilesDownloader/Private/RuntimeChunkDownloader.cpp

```diff
@@ -20,7 +20,7 @@ TFuture<FRuntimeChunkDownloaderResult> FRuntimeChunkDownloader::DownloadFile(con
 	if (bCanceled)
 	{
 		UE_LOG(LogRuntimeFilesDownloader, Warning, TEXT("Canceled file download from %s"), *URL);
-		return MakeFulfilledPromise<FRuntimeChunkDownloaderResult>(FRuntimeChunkDownloaderResult{EDownloadToMemoryResult::Cancelled, TArray64<uint8>()}).GetFuture();
+		return MakeFulfilledPromise<FRuntimeChunkDownloaderResult>(FRuntimeChunkDownloaderResult{EDownloadToMemoryResult::Cancelled, {}, {}}).GetFuture();
```

Comment:
Many `{}` replacing `TArray64<uint8>()`, is this to reduce memory allocations for unused empty arrays? IF so, `{}` is accepted in place?

Reply:
Nah, {} is just syntactic sugar in cpp for "empty $thing". I think it uses the std::initializer_list. Just a lot less noise to type and read, but has the same effect - we still need to pass a valid TArray64, since this isn't a pointer or anything